Gopkg.lock

```diff
@@ -124,6 +124,32 @@
   revision = "1624edc4454b8682399def8740d46db5e4362ba4"
   version = "v1.1.5"
 
+[[projects]]
+  branch = "master"
+  digest = "1:d6ec8799b1f86397187016bfc7e129e51726e9b46106b0ca52f5478f4279a4e8"
+  name = "github.com/kubeflow/mpi-operator"
```

Comment:
This doesn't look right. It shouldn't depend on itself.

Reply:
@jq I think you need to do a `dep ensure` again and push your new changes in `Gopkg.lock` which should resolve this issue.

Reply:
remove it directly